### PR TITLE
JCOREX:-352 Eletter module update hook 

### PR DESCRIPTION
**Pull request by akashnegi07**

### CODE REVIEW

File: web/modules/highwire/highwire_site_style/highwire_site_style.install [hunk 1]
```diff
@@ -56,3 +56,16 @@ function highwire_site_style_update_9003(){
...
   }  
 }
+
+/**
+ * Implements hook_update()
+ *
+ * Perform actions to update values to Eletter module.
+ */
+function highwire_site_style_update_9004(){
+  /* For trendmd*/
```
Comment (RahulSatija49) [inline web/modules/highwire/highwire_site_style/highwire_site_style.install:66]: check comment, showing trendmd

File: web/modules/highwire/highwire_site_style/highwire_site_style.install [hunk 2]
```diff
@@ -56,3 +56,16 @@ function highwire_site_style_update_9003(){
...
+  $Eletter_enable = \Drupal::service('module_installer')->install(['highwire_eletter'], TRUE); 
+  if (!$Eletter_enable) {
+	  throw new DrupalUpdateException('Eletter module not enabled.');
```
Comment (RahulSatija49) [inline web/modules/highwire/highwire_site_style/highwire_site_style.install:69]: Check alignment

**Review by RahulSatija49 — COMMENTED** [review]
Please check both points

File: web/modules/highwire/highwire_site_style/highwire_site_style.install (hunk 1 above)
Comment (akashnegi07) [reply web/modules/highwire/highwire_site_style/highwire_site_style.install]: This comment issue is fixed now.

**Pull request merged**